app/src/interfaces/_system/system-filter/system-filter.vue

```diff
@@ -42,6 +42,7 @@
 					v-if="collectionRequired"
 					:collection="collection"
 					include-functions
+					:exclude-relations="includeValidation"
```

Comment:
I think this should be a separate prop coming in to the interface as I'm sure there will be system-filter uses outside of validation where we also need to limit relational fields (like conditions).

Reply:
Refactored to being includeRelations